examples/plot_fit_fake_data.py

```diff
@@ -0,0 +1,37 @@
+"""
+Example for fitdata function
+============================
+This example creates fake data with noise
+then fits the exponential with the fitdata
+function.
+"""   
+from pyspecdata import *
+from pyspecdata import fitdata 
+from sympy import symbols
+import numpy as np
+# %%
+# creating a fake data recovery curve
+tau = nddata(r_[0:2:100j], 'tau')
+fake_data = 102*(1-2*exp(-tau*6.0))
+fake_data.add_noise(5.0)
+
+# %%
+# fitting data
+f = fitdata(fake_data)
+M0,Mi,R1,vd = sympy.symbols("M_0 M_inf R_1 tau",real=True)
+f.functional_form = Mi + (M0-Mi)*sympy.exp(-vd*R1)
```

Comment:
didn't we have a script that did this already?  Why has this been created as an entirely new file??

Reply:
yes, there is now one example only.